modular_skyrat/master_files/code/modules/client/preferences_savefile.dm

```diff
@@ -117,6 +117,42 @@
 			// You weren't meant to be able to pick sizes over this anyways.
 			write_preference(GLOB.preference_entries[/datum/preference/choiced/breasts_size], GLOB.breast_size_translation["[min(old_breast_prefs, 10)]"])
 
+	if(current_version < 3)
```

Comment:
I feel like this will cause issue as soon as we give JSON savefiles a shot, so I'm probably going to have to get that in before we get this in.

Reply:
Ping me at least ten times when that gets done, and I'll get around to suffering through mutilating everyone's savefiles for the fourth time.